.gitpod.yml

```diff
@@ -5,24 +5,20 @@ tasks:
   - name: Unit Test Coverage
     init: gp sync-await installation
     command: |
-      npm run test:unit
-      cd coverage/lcov-report/
-      http-server -p 8000 &
-      cd ../../
```

Comment:
Let's keep this in here. The idea is that the developer can immediately see the projects test coverage when the environment is boot up.

Reply:
Thank you for clarifying, I've undone these lines.